ui/lib/components/CopyLink/index.tsx

```diff
@@ -1,24 +1,39 @@
 import React, { useState } from 'react'
 import { CopyToClipboard } from 'react-copy-to-clipboard'
-import { addTranslationResource } from '@lib/utils/i18n'
 import { useTranslation } from 'react-i18next'
 import { useTimeoutFn } from 'react-use'
+import { Tooltip } from 'antd'
 import { CheckOutlined, CopyOutlined } from '@ant-design/icons'
+import { addTranslationResource } from '@lib/utils/i18n'
 
 import styles from './index.module.less'
 
+type DisplayVariant = 'default' | 'original_sql' | 'formatted_sql'
+const transKeys: { [K in DisplayVariant]: string } = {
+  default: 'copy',
+  original_sql: 'copyOriginal',
+  formatted_sql: 'copyFormatted',
+}
+
 export interface ICopyLinkProps {
   data?: string
+  displayVariant?: DisplayVariant
 }
 
 const translations = {
   en: {
-    text: 'Copy',
+    copy: 'Copy',
+    copyOriginal: 'Copy Original',
+    copyFormatted: 'Copy Formatted',
     success: 'Copied',
+    tooltip: 'Copy by this button if you need to run it in SQL client',
   },
   zh: {
-    text: '复制',
+    copy: '复制',
+    copyOriginal: '复制原始 SQL',
+    copyFormatted: '复制格式化 SQL',
     success: '已复制',
+    tooltip: '如果你需要在 SQL 客户端执行此 SQL 语句，请使用这个按钮进行复制',
```

Comment:
This tooltip can cause misleading because it is describing a bug.

Reply:
😂